src/website/app/demos/ThemeProvider/components/Sample.js

```diff
@@ -16,21 +16,16 @@
 
 /* @flow */
 import React from 'react';
-import { createStyledComponent } from '../../../../../utils';
-
-type Props = {
-  className?: string,
-  text?: string
-};
+import { createStyledComponent } from '../../../../../styles';
 
 const Root = createStyledComponent('span', ({ theme }) => ({
   backgroundColor: theme.Sample_backgroundColor || 'lavender',
-  color: theme.Sample_color || theme.color_text,
+  color: theme.Sample_color || theme.color_primary,
```

Comment:
It is used in a lot of examples.  I probably updated it to match the example that I was working with rather than correcting the example.  Will update.